src/metabase/sync/sync_metadata/sync_table_privileges.clj

```diff
@@ -14,10 +14,7 @@
    This is a cache of the data returned from `driver/table-privileges`, but it's stored in the database for performance."
   [database :- (mi/InstanceOf :model/Database)]
   (let [driver (driver.u/database->driver database)]
-    (when (and (not= :redshift driver)
-               ;; redshift does support table-privileges, but we don't want to sync it now because table privileges are
-               ;; meant to enhance action features, but redshift does not support actions for now, so we skip it here.
-               (driver/database-supports? driver :table-privileges database))
+    (when (driver/database-supports? driver :table-privileges database)
```

Comment:
This is not really related but I just saw it and cleaned it up. Redshift no longer supports table-privileges as of https://github.com/metabase/metabase/pull/41068